test/reporter.spec.js

```diff
@@ -154,7 +154,7 @@ describe('#reporter()', function () {
 
       stream.end()
 
-      expect(gulpUtil.log.called)
+      expect(console.log.called)
```

Comment:
I'm surprised this worked, since you were calling `console.error` above. Did you run the unit tests before committing this code? Run `npm test` and see what you get.

Reply:
<img width="727" alt="screen shot 2018-08-03 at 6 24 45 pm" src="https://user-images.githubusercontent.com/21992321/43668362-89d1781a-974a-11e8-852a-937947203e86.png">
Terminal Output attached. I believe the only reason this works is because of the commented `expect` statement above. I need to research further into your test script to understand what it is actually doing with the `gulpUtil` variable.